src/dsp_tools/commands/xmlupload/check_consistency_with_ontology.py

```diff
@@ -13,6 +15,8 @@
 knoraUndeclared: Pattern[str] = regex.compile(r"^\w+$")
 genericPrefixedOntology: Pattern[str] = regex.compile(r"^[\w\-]+:\w+$")
 
+cwd = os.getcwd()
```

Comment:
If it's not absolutely necessary, this shouldn't be global. (And here, I don't see a necessity)